e2e/update_pnpm_lock/test.sh

```diff
@@ -54,8 +54,8 @@ ASPECT_RULES_JS_DISABLE_UPDATE_PNPM_LOCK=
 _sedi 's#"@types/node": "16"#"@types/node": "14"#' package.json
 
 # Trigger the update of the pnpm lockfile
-if ! bazel run "$BZLMOD_FLAG" @npm//:sync; then
```

Comment:
Does `bazel run @npm//:sync` not work under bzlmod??

Reply:
It should work fine. I think I could revert this actually, I must have been doing something weird to go down the path of changing this (e2e tests that modify source are odd...) 🤔